src/azure-cli/azure/cli/command_modules/aro/_help.py

```diff
@@ -7,66 +7,66 @@
 
 
 helps['aro'] = """
-    type: group
-    short-summary: Manage Azure Red Hat OpenShift clusters.
+  type: group
+  short-summary: Manage Azure Red Hat OpenShift clusters.
```

Comment:
Are these changes all whitespace? Are they needed or can we exclude them from the patch?

Reply:
| Please can you fix the whitespace diffs in _help.py via this PR as well?

@ehashman, Yes, they are.  @jim-minter requested above that we align these with our `_help.py`.